clickstream-event-interceptor/src/main/java/clickstream/interceptor/InterceptedEvent.kt

```diff
@@ -0,0 +1,88 @@
+package clickstream.interceptor
+
+/**
+ * Intercepted event from clickstream with different states.
+ * [Instant] events : fire and forget.
+ * [Scheduled] events : Events that are [Scheduled] -> [Dispatched] -> [Acknowledged]
+ *
+ * @property eventId
+ * @property eventName
+ * @property productName
+ * @property timeStamp
+ * @constructor Create empty Intercepted event
+ */
+public sealed class InterceptedEvent(
```

Comment:
Same. We don't use this class anywhere else other than the interface.. Should we remove it?